src/index.js

```diff
@@ -82,4 +84,8 @@ function delegateCommand() {
   });
 
   shutdown(res.status);
+}
+
+(async function () {
+  await main();
 })();
```

Comment:
why this change? I don't see any benefit

Reply:
it's more readable i believe, we would like to see "main" so we know where the code is being executed.

doesn't hurt

Reply:
yes, just readability 
Javascript is not easy to read